### DISCUSSION TITLE
CLI command for outputting current config setting values

### DISCUSSION THREAD

**Discussion by PatrickMessierGoC** [craftcms/cms#8899] [Ideas]
### Description
Scanning all configuration files and searching for default values from web documentation is a long process when troubleshooting. A nice solution would be to extend the Craft CLI tool to dump the value of all CraftCMS configuration values for debugging purposes.

Ex: ./craft config/dump [options] 
--all : dump all configuration variable values 
--list [comma or semi colon separated list of configuration variable names] :  dump the listed configuration variable values. 

Output Example.
[configuration variable name 1]: [value 1]
[configuration variable name 2]: [value 2]
...
[configuration variable name N]: [value N]

BONUS Points: 
Sort the command output in alphabetical order of configuration variable name.

**Comment by brandonkelly** [comment]
To be clear, are you referring to the [general config settings](https://craftcms.com/docs/3.x/config/config-settings.html)?

**Comment by PatrickMessierGoC** [comment]
@brandonkelly Yes

**Comment by PatrickMessierGoC** [comment]
@brandonkelly Should I conclude that this request is "In the development queue"?

**Comment by brandonkelly** [comment]
Not planning on implementing it at this time, but open to adding it at some point, especially if the request picks up some traction.
